docs/adapters/fauna.md

```diff
@@ -12,13 +12,13 @@ You can find the Fauna schema and seed information in the docs at [next-auth.js.
 ## Getting Started
 
 :::warning
-When using the **NextAuth v4 beta**, please make sure to use the `next` tagged version of your adapter. For more info on adapter changes, see [the migration docs](/getting-started/upgrade-v4#adapters)
+When using the **NextAuth v4**, please make sure to use the `next` tagged version of your adapter. For more info on adapter changes, see [the migration docs](/getting-started/upgrade-v4#adapters)
```

Comment:
All these warnings can be removed! Adapters will go stable as well :slightly_smiling_face: